providers/storage/tableMappingStore.js

```diff
@@ -55,8 +55,8 @@ class AzureTableMappingStore {
     return this.baseStore.list(type);
   }
 
-  delete(type, url) {
-    return this.baseStore.delete(type, url);
+  delete(type, key) {
+    return this.baseStore.delete(type, key);
```

Comment:
consider deleting both by url or urn and handle rejection if the base store is keyed by the "other" key